README.md

```diff
@@ -124,3 +124,22 @@ synchronization:
   roles:
     - linux-system-roles.timesync
 ```
+
+Install and configure chrony with multiple NTP servers and custom advanced 
+settings: log ```measurements```,```statistics``` and ```tracking```
+into ```/var/log/chrony```:
+
+
+```yaml
+- hosts: targets
+  vars:
+    timesync_ntp_servers:
+      - hostname: foo.example.com
+      - hostname: bar.example.com
+      - hostname: baz.example.com
+    timesync_chrony_custom_settings:
```

Comment:
please document `timesync_chrony_custom_settings` in the README.md in the Role Variables section - https://github.com/linux-system-roles/timesync/blob/ccc910ca57e34c6b79ca455e1c33c32df92f3bd9/README.md#role-variables